src/client.js

```diff
@@ -3,8 +3,26 @@ import React from 'react';
 /*eslint-enable no-unused-vars*/
 import { render } from 'react-dom';
 import { Router, browserHistory } from 'react-router';
+import { IntlProvider, addLocaleData } from 'react-intl';
 import routes from './routes.js';
 import ga from 'react-ga';
+var locales = require('../public/locales.json');
+
+function createElement(Component, props) {
```

Comment:
This change is a standard way to inject the react-intl context into our pages.

It also adds the locale data, messages, and locale code.

locale data is specific to the language used, and is meta data. Things like, language formatting, characters used in that language, stuff like that.

messages is the ke values for the stings. So when we want to swap out "submit_button_text" key to be "Submit Now" this info is stored in messages.

locale is just "en-US", "pt-BR", "de", "es", etc.